charmcraft/commands/build.py

```diff
@@ -23,16 +23,12 @@
 import zipfile
 from typing import List, Optional
 
-from charmcraft import linters, parts
+from charmcraft import env, linters, parts
 from charmcraft.bases import check_if_base_matches_host
 from charmcraft.cmdbase import BaseCommand, CommandError
 from charmcraft.config import Base, BasesConfiguration, Config
 from charmcraft.deprecations import notify_deprecation
-from charmcraft.env import (
-    get_managed_environment_home_path,
-    get_managed_environment_project_path,
-    is_charmcraft_running_in_managed_mode,
-)
+from charmcraft.env import is_charmcraft_running_in_managed_mode
```

Comment:
Why not call this one also through `env.`?

Reply:
No specific reason, will update to call using `env.`.